<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bottom</class>
 <widget class="QWidget" name="Bottom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>353</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="qtabwidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="metamath">
      <attribute name="title">
       <string>Metamath</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="metamathTextEdit">
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="metamathExecuteLabel">
           <property name="text">
            <string>Command:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="metamathCommandComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="metamathExecuteButton">
           <property name="toolTip">
            <string>execute command</string>
           </property>
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="metamathClearButton">
           <property name="toolTip">
            <string>clear console</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="russell">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Russell</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPlainTextEdit" name="russellTextEdit">
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="russellExecuteLabel">
           <property name="text">
            <string>Command:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="russellCommandComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="russellExecuteButton">
           <property name="toolTip">
            <string>execute command</string>
           </property>
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="russellClearButton">
           <property name="toolTip">
            <string>clear console</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
